tests/testAutopas/C08TraversalTest.cpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * C08TraversalTest.cpp
+ *
+ *  Created on: 5/24/18
+ *     Aauthor: F. Gratl
+ */
+
+
+#ifdef AUTOPASAUTOPAS_OPENMP
+#include <omp.h>
+#endif
+#include "C08TraversalTest.h"
+
+using ::testing::_;
+using ::testing::AtLeast;
+
+void C08TraversalTest::fillWithParticles(
```

Comment:
could you add this to tests/testAutoPas/testingHelpers/
as GridGenerator instead? we are probably reusing this a lot...
(cf. Verlet-List-Vectorization)

Reply:
done b7e601d098ccf1e3698f19afd0f23a93ea836307